Tests/ReviewComponentTest.js

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
```

Comment:
Is this file and Tests/test.js the same / similar?
Is one a more updated version of the other?

Reply:
Yes, it's a duplicate. Trying to figure out if/how to use multiple test files.